lib/usbmux/index.js

```diff
@@ -61,13 +62,28 @@ function swap16 (val) {
  * @throws {B.TimeoutError} if connection timeout happened
  * @returns {Promise<NodeJS.Socket>} Connected socket instance
  */
-async function getDefaultSocket (opts = {}) {
-  const {
-    socketPath = DEFAULT_USBMUXD_SOCKET,
-    socketPort = DEFAULT_USBMUXD_PORT,
-    socketHost = DEFAULT_USBMUXD_HOST,
-    timeout = 5000,
-  } = opts;
+async function getDefaultSocket(opts = {}) {
+  const defaults = {
+    socketPath: DEFAULT_USBMUXD_SOCKET,
+    socketPort: DEFAULT_USBMUXD_PORT,
+    socketHost: DEFAULT_USBMUXD_HOST,
+    timeout: 5000
+  };
+
+  if (process.env.USBMUXD_SOCKET_ADDRESS && !opts.socketPath && !opts.socketPort && !opts.socketHost) {
+    log.debug(`Using USBMUXD_SOCKET_ADDRESS environment variable as default socket: ${process.env.USBMUXD_SOCKET_ADDRESS}`);
+    // "unix:" or "UNIX:" prefix is optional for unix socket paths.
+    const usbmuxdSocketAddress = process.env.USBMUXD_SOCKET_ADDRESS.replace(/^(unix|UNIX):/i, '');
```

Comment:
`(unix|UNIX)` some part of the regexp is probably redundant if it already has a case insensitive flag.

Reply:
You're right, `/i` disables the sensitivity already. Updated.